DTP_config.py

```diff
@@ -1,17 +1,17 @@
 # -*- coding: utf-8 -*-`
 
 
-# Copyright Inria Sophia Antipolis-Méditerranée 2022. All Rights Reserved.
```

Comment:
The name in fact changed and we are now called differently. I will fix that in another commit.